examples/amp-story/player.html

```diff
@@ -2,6 +2,7 @@
   <head>
     <title>Story Player (Non-AMP)</title>
     <script async src="../../dist/amp-story-player.js"></script>
+    <link href="../../build/css/amp-story-player-out.css" rel='stylesheet' type='text/css'>
```

Comment:
Is it possible to get a URL that works on our machines during development, and that we can host on firebase using the existing scripts?
What's going to be the final production URL to load this stylesheet?

Reply:
I don't think we need to do anything to make this work in Firebase?  As it's written, it should still work.  For example, right now, I can hit http://stamp-gmajoulet.firebaseapp.com/dist/video-autoplay-out.css.  I would still expect the relative URLs to work just fine, even on Firebase.

Reply:
Oh nice! I should've checked this before 😅